src/mcp_gateway/approval/notifier.py

```
@@ -0,0 +1,75 @@
+
+    session_id: str
+    agent_id: str
+    intent: str
+    tool_name: str
+    arguments: Mapping[str, Any]
+    requested_at: datetime = Field(default_factory=lambda: datetime.now(UTC))
+
+    @field_validator("arguments", mode="after")
+    @classmethod
+    def _make_immutable(cls, v: Mapping[str, Any]) -> Mapping[str, Any]:
+        """引数を不変（MappingProxyType）に変換します。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)